src/pods/canvas/components/relation/components/clickeable-self.component.tsx

```diff
@@ -0,0 +1,39 @@
+import { Coords, GUID } from '@/core/model';
+import { TABLE_CONST } from '@/core/providers';
+import classes from './clickable-path.component.module.css';
+
+interface Props {
+  id: GUID;
+  startCoords: Coords;
+  endCoords: Coords;
+  onClick: (relationId: GUID) => void;
+  onDoubleClick: (relationId: GUID) => void;
+}
+
+export const ClickableSelfComponent: React.FC<Props> = props => {
```

Comment:
What's this? is this related to export case? I guess you commented you were going to fix something else

Reply:
I commented to you, I went to fix this